x/cronos/keeper/precompiles/bank.go

```diff
@@ -18,43 +20,69 @@ import (
 )
 
 const (
-	EVMDenomPrefix          = "evm/"
-	BankContractRequiredGas = 10000
+	EVMDenomPrefix      = "evm/"
+	MintMethodName      = "mint"
+	BurnMethodName      = "burn"
+	BalanceOfMethodName = "balanceOf"
+	TransferMethodName  = "transfer"
 )
 
 var (
-	bankABI             abi.ABI
-	BankContractAddress = common.BytesToAddress([]byte{100})
+	bankABI                 abi.ABI
+	bankContractAddress     = common.BytesToAddress([]byte{100})
+	bankGasRequiredByMethod = map[[4]byte]uint64{}
 )
 
 func init() {
 	if err := bankABI.UnmarshalJSON([]byte(bank.BankModuleMetaData.ABI)); err != nil {
 		panic(err)
 	}
+	for methodName := range bankABI.Methods {
```

Comment:
could make gosec happy here

Reply:
the gosec alerts is about the randomness to run over a map right?

not sure how to make it happy

Reply:
just range warning, btw I see failed to create client error after fee adjust

Reply:
I am suspecting it is a gas issue. 

The go-relayer uses cosmos estimation multiply by an adjustment coef
https://github.com/crypto-org-chain/relayer/blob/main/relayer/chains/cosmos/tx.go#L308C1-L308C1

I have tried using a high adjustement coefficient but no luck


Strangly the test pass locally....

Reply:
can you fix relayer [config](https://github.com/crypto-org-chain/cronos/blob/78e6423e88295be260751e68313306d8dc563ea2/integration_tests/configs/ibc.jsonnet#L136-L137)